src/app/shared/capacitor-plugins/mock-app-plugin.ts

```diff
@@ -20,35 +20,6 @@ export class MockAppPlugin implements AppPlugin {
     throw new Error('Method not implemented.');
   }
 
-  // addListener(
```

Comment:
In the previous commit, I forgot to remove this commented code. It's safe to remove these lines.